torch/_subclasses/fake_tensor.py

```diff
@@ -378,13 +379,15 @@ def mk_fake_tensor(make_meta_t):
                     constant=t if make_constant else None,
                 )
 
-        out = self.meta_converter(
-            t,
-            shape_env=shape_env,
-            callback=mk_fake_tensor,
-            source=source,
-            symbolic_context=symbolic_context,
-        )
+        disable_functorch = torch._C._DisableFuncTorch
+        with disable_functorch():
```

Comment:
Hmm, this might be concerning (that we need to turn off functorch at random points in Dynamo). let me think about this some more.

Reply:
If we don't turn off functorch and grad is enabled, any torch.* op in this file returns a GradTrackingTensor.

Reply:
Maybe let's try to fix the TensorWrapper fake-ification (that I mentioned in the other comments) and see if this is still necessary.